send-data/syslog-proxy.mdx

```diff
@@ -1,79 +1,122 @@
 ---
-title: 'Send data from Syslog to Axiom'
-description: 'Learn how to monitor syslog events on your devices and send them directly into Axiom'
+title: 'Send data from syslog to Axiom'
+description: 'This page explains how to send data from a syslog logging system to Axiom.'
 sidebarTitle: Syslog
 tags:
   ['axiom documentation', 'documentation', 'axiom', 'guide', 'syslog', 'syslog logs', 'tcp', 'udp', 'docker']
 ---
 
-Logs are generated on all Network devices, Axiom Syslog Proxy ships logs to Axiom, acting as a Syslog server.
+import Prerequisites from "/snippets/standard-prerequisites.mdx"
+import Replacement from "/snippets/standard-replacement.mdx"
 
-With Syslog logging system, you can monitor events on your devices and send them directly into Axiom, this also helps you retain your data logs.
+The Axiom Syslog Proxy acts as a syslog server to send data to Axiom.
 
-The UDP log messages are sent on `UDP` port `514` to the Syslog server.
+<Note>
+    The Axiom Syslog Proxy is an open-source project and welcomes your contributions. For more information, see the [GitHub repository](https://github.com/axiomhq/axiom-syslog-proxy).
+</Note>
 
-The TCP log messages are sent on `TCP` port `601` to the Syslog server.
+## Syslog limitations and recommended alternatives
 
-## Installation
+Syslog is an outdated protocol from the 1980s. Some of the limitations are the following:
 
-### Download the pre-compiled and archived binary manually
+- Lack of error reporting and feedback mechanisms when issues occur.
+- Inability to gracefully terminate the connection. This can result in missing data.
 
-Binary releases are available on [GitHub Releases](https://github.com/axiomhq/axiom-syslog-proxy/releases/latest).
+<Note>
+For a more reliable and modern logging experience, consider using tools like [Vector](https://vector.dev/) to receive syslog messages and [forward them to Axiom](/send-data/vector). This approach bypasses many of syslog’s limitations.
+</Note>
 
-### Install using [Homebrew](https://brew.sh)
+<Prerequisites />
+
+Other requirements:
+
+- **Message size limit:** Axiom currently enforces an 8KB per-message size limit. This is in line with RFC5425 guidelines. Any message exceeding the limit causes the connection to close because Axiom doesn’t support ingesting truncated messages.
```

Comment:
```suggestion
- **Message size limit:** Axiom currently enforces an 64KB per-message size limit. This is in line with RFC5425 guidelines. Any message exceeding the limit causes the connection to close because Axiom doesn’t support ingesting truncated messages.
```

This was increased by @gordallott recently.

Reply:
Cool, thanks. I thought this was still in planning phase.